lib/stm32/l4/rcc.c

```diff
@@ -509,7 +626,8 @@ uint32_t rcc_get_i2c_clk_freq(uint32_t i2c)
 /** @brief Get the peripheral clock speed for the SPI device at base specified.
  * @param spi  Base address of SPI device to get clock frequency for (e.g. SPI1_BASE).
  */
-uint32_t rcc_get_spi_clk_freq(uint32_t spi) {
+uint32_t rcc_get_spi_clk_freq(uint32_t spi)
+{
```

Comment:
You're right, this is wrongly formatted, but it's unrelated to your code, and so generally you should not include this.  It makes it easier to apply just your change, to only touch your code.  I can handle this for you though.